### PR TITLE
docs: use sentence casing for sections

### PR DESCRIPTION
**Pull request by moldy530**
# Pull Request Checklist

- [ ] Did you add new tests and confirm existing tests pass? (`yarn test`)
- [ ] Did you update relevant docs? (docs are found in the `site` folder, and guidelines for updating/adding docs can be found in the [contribution guide](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md))
- [ ] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples)? (e.g. `feat!: breaking change`)
- [ ] Did you run lint (`yarn lint:check`) and fix any issues? (`yarn lint:write`)
- [ ] Did you follow the [contribution guidelines](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md)?

---

## PR-Codex overview
This PR updates the titles and headings in various markdown files to use consistent capitalization. 

### Detailed summary
- Updated titles and headings in `faqs.md`, `contact-us.md`, `index.md`, `light-account/index.md`, `custom-signer.md`, `nft.md`, `gas-benchmarks.md`, `simulate-user-operations.md`, `light-account.md`, `session-keys/index.md`, `index.md`, `gas-manager.md`, `demos.md`, `magic/introduction.md`, `turnkey/introduction.md`, `contributing.md`, `send-user-operations.md`, and `checking-eligibility.md`.

> The following files were skipped due to too many changes: `site/using-smart-accounts/sponsoring-gas/checking-eligibility.md`, `site/getting-started/setup.md`, `site/signers/contributing.md`, `site/.vitepress/sidebar/index.ts`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: site/.vitepress/sidebar/index.ts [hunk 1]
```diff
@@ -78,11 +78,11 @@ export const sidebar: DefaultTheme.Sidebar = [
         collapsed: true,
         items: [
           {
-            text: "Using Your Own",
+            text: "Use your own",
             link: "/guides/custom-signer",
           },
           {
-            text: "Contributing Your Signer",
+            text: "Contribute your Signer",
```
Comment (denniswon) [inline site/.vitepress/sidebar/index.ts:85]:
```suggestion
            text: "Contribute your signer",
```

File: site/overview/demos.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ head:
 
 # Demos
 
-## Gasless Token Minting Demo
+## Gasless Token minting demo
```
Comment (denniswon) [inline site/overview/demos.md:17]:
```suggestion
## Gasless token minting demo
```

File: site/signers/contributing.md [hunk 3]
```diff
@@ -18,11 +18,11 @@ head:
       content: Follow this guide to contribute a new Signer to Account Kit documentation, and enable developers to sign ERC-4337 transactions with your Signer.
 ---
 
-# Contributing Your Signer
+# Contribute your Signer
```
Comment (denniswon) [inline site/signers/contributing.md:21]:
```suggestion
# Contribute your signer
```

File: site/signers/contributing.md [hunk 4]
```diff
@@ -38,13 +38,13 @@ If you your Signer implementation requires adding SDKs as dependencies, you shou
 If your SDK is based on a frontend Javascript framework, such as React.js or Vue.js, you will just have to follow step 2 onwards to submit documentation with an example snippet clarifying that your Signer must be used in said framework.
 :::
 
-## 2. Adding Documentation about Your Signer
+## 2. Add documentation about your Signer
```
Comment (denniswon) [inline site/signers/contributing.md:41]:
```suggestion
## 2. Add documentation about your signer
```

File: site/.vitepress/sidebar/index.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (moldy530) [reply site/.vitepress/sidebar/index.ts:85]: yea Signer is a proper noun imo

Comment (moldy530) [reply site/.vitepress/sidebar/index.ts:85]: same here, signer is a proper noun like Smart Account

File: site/signers/contributing.md [hunk 5]
```diff
@@ -18,11 +18,11 @@ head:
       content: Follow this guide to contribute a new Signer to Account Kit documentation, and enable developers to sign ERC-4337 transactions with your Signer.
 ---
 
-# Contributing Your Signer
+# Contribute your Signer
 
 If you'd like to add your Signer to Account Kit, we welcome PRs! You'll need to fork the [`aa-sdk` Github repo](https://github.com/alchemyplatform/aa-sdk) and then follow the below steps.
 
-## 1. Add Your Signer to [`aa-signers`](https://github.com/alchemyplatform/aa-sdk/tree/main/packages/signers)
+## 1. Add your signer to [`aa-signers`](https://github.com/alchemyplatform/aa-sdk/tree/main/packages/signers)
```
Comment (denniswon) [inline site/signers/contributing.md:25]:
```suggestion
## 1. Add your Signer to [`aa-signers`](https://github.com/alchemyplatform/aa-sdk/tree/main/packages/signers)
```

**Review by denniswon — APPROVED** [review]
lgtm

**Pull request merged**